js/react_native/e2e/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionSha256Sum=7faa7198769f872826c8ef4f1450f839ec27f0b4d5d1e51bade63667cbccd205
```

Comment:
need to update the sha256sum as well
https://gradle.org/release-checksums/

could we use gradle-7.3.3-bin.zip instead? it's a smaller download

Reply:
Sorry about that I wasn't realize of the checksum because to it's not failed on my local env, I'll fix it and change to bin. Thanks!